master/docs/developer/database.rst

```diff
@@ -1779,16 +1799,22 @@ Postgres, add them to ``implied_indexes`` in
 
 Foreign key checking
 --------------------
-Non sqlite db backends are checking the foreign keys consistancy. As sqlite is much
-easier to install, most of the developer would only test against sqlite, and then get
-errors on metabuildbot. In order to avoid that the fakedb can check the foreign key
-consistancy of your test data. for this, just enable it with::
+All supported db backends are checking the foreign keys consistancy.
```

Comment:
Ah you are right. You can run buildbot with Innodb, but you have to configure it for accepting bigger keys. I made a PoC once, but never got much further.
We need to document and test it more heavily